src/librustc_trans/trans/cabi_arm_ios.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright 2012-2013 The Rust Project Developers. See the COPYRIGHT
```

Comment:
How much of this file is a duplicate of `cabi_arm`? Would it be possible to delegate non-special cases into `cabi_arm` where possible?